install_config/install/deploy_router.adoc

```diff
@@ -450,10 +456,114 @@ command when
 link:../../admin_guide/high_availability.html#configuring-a-highly-available-routing-service[creating
 a highly-available routing service].
 
+== Deploying the F5 Router
+
+ifdef::openshift-enterprise[]
+[NOTE]
+====
+The F5 router plug-in is available starting in OpenShift Enterprise 3.0.2.
+====
+endif::[]
+
+The F5 router plug-in is provided as a Docker image and run as a pod, just like
+the link:#haproxy-router[default HAProxy router]. Deploying the F5 router is
+done similarly as well, using the `oadm router` command but providing additional
+flags (or environment variables) to specify the following parameters for the *F5
+BIG-IP®* host:
+
+[[f5-router-flags]]
+[cols="1,4"]
+|===
+|Flag |Description
+
+|`--type=f5-router`
+|Specifies that an F5 router should be launched (the default `--type` is
+*haproxy-router*).
+
+|`--external-host`
+|Specifies the *F5 BIG-IP®* host's management interface's host name or IP
+address.
+
+|`--external-host-username`
+|Specifies the *F5 BIG-IP®* user name (typically *admin*).
+
+|`--external-host-password`
+|Specifies the *F5 BIG-IP®* password.
+
+|`--external-host-http-vserver`
+|Specifies the name of the F5 virtual server for HTTP connections.
+
+|`--external-host-https-vserver`
+|Specifies the name of the F5 virtual server for
+HTTPS connections.
+
+|`--external-host-private-key`
+|Specifies the path to the SSH private key file for the *F5 BIG-IP®* host.
+Required to upload and delete key and certificate files for routes.
+
+|`--external-host-insecure`
+|A Boolean flag that indicates that the F5 router should skip strict certificate
+verification with the *F5 BIG-IP®* host.
+|===
+
+As with the HAProxy router, the `oadm router` command creates the service and
+deployment configuration objects, and thus the replication controllers and
+pod(s) in which the F5 router itself runs. The replication controller restarts
+the F5 router in case of crashes. Because the F5 router is only watching routes
+and endpoints and configuring *F5 BIG-IP®* accordingly, running the F5 router in
+this way along with an appropriately configured *F5 BIG-IP®* deployment should
+satisfy high-availability requirements.
+
+To deploy the F5 router:
+
+. First,
+link:../../admin_guide/routing_from_edge_lb.html#establishing-a-tunnel-using-a-ramp-node[establish
+a tunnel using a ramp node], which allows for the routing of traffic to pods
```

Comment:
Not sure if we should in-line all those steps in this topic, or leave it in the other topic and link (as shown here).